Packs/Base/Scripts/SearchIndicatorRelationships/SearchIndicatorRelationships.py

```diff
@@ -51,36 +51,65 @@ def to_context(relationships: list, verbose: bool) -> List[dict]:
     return context_list
 
 
-def handle_stix_types(entities_types):
+def handle_stix_types(entities_types: str) -> str:
     entities_types = argToList(entities_types)
     for e_type in entities_types:
         FeedIndicatorType.indicator_type_by_server_version(e_type)
     return ','.join(str(x) for x in entities_types).replace(', ', ',')
 
 
+def search_relationships_6_6_0(args: dict) -> Optional[List[dict]]:
+    """
+    XSOAR Version 6.6.0 requires quite different handling for the call to `searchRelationships` server command:
+    - The relationships args are provided under the 'filter' key.
+    - The 'entities', 'entityTypes' and 'relationshipNames' keys are given as a list of strings,
+      rather than a comma-separated list.
+    - An exception is raised in case of error.
+    - The returned value is the data directly, and not an entry.
+    """
+
+    for list_arg in ['entities', 'entityTypes', 'relationshipNames']:
+        args[list_arg] = argToList(args[list_arg]) if args[list_arg] else None
+    res = demisto.executeCommand("searchRelationships", {'filter': args})
```

Comment:
You should use: `demisto.searchRelationships` (it uses internally executeCommand, but it may change...)